tests/support/server.rs

```diff
@@ -55,10 +56,16 @@ pub fn spawn(txns: Vec<Txn>) -> Server {
             }
 
             if let Some(dur) = txn.write_timeout {
-                let headers_end = ::std::str::from_utf8(&reply).unwrap().find("\r\n\r\n").unwrap() + 4;
+                let headers_end = unsafe { ::std::str::from_utf8_unchecked(&reply) }.find("\r\n\r\n").unwrap() + 4;
```

Comment:
The fact that it is UB doesn't change, and this code may break any time. Change the type signature instead.